lib/dateTime.js

```diff
@@ -763,7 +775,7 @@ export function getDateTimeDescriptor() {
 			];
 			break;
 		case 'es':
-			dateFormats = ['dddd d\' de \'MMMM\' de \'yyyy', 'd\' de \'MMMM\' de \'yyyy', 'd/M/yyyy', 'MMMM yyyy', 'd\' de \'MMMM', 'd\' de \'MMM'];
+			dateFormats = ['dddd d\' de \'MMMM\' de \'yyyy', 'd\' de \'MMMM\' de \'yyyy', 'dd/MM/yyyy', 'MMMM yyyy', 'd\' de \'MMMM', 'd\' de \'MMM'];
```

Comment:
There was a slight deviation here in the base `es` data we had, but now that the monolith has `es-ES` and it was the same as `es-MX` I've combined them.